templates/parachain/README.md

```diff
@@ -1,22 +1,108 @@
+
+A Polkadot SDK based project such as this one consists of:
+
+- 💿 a [Node](./node/README.md) - the binary application.
+- 🧮 the [Runtime](./runtime/README.md) - the core logic of the parachain.
+- 🎨 the [Pallets](./pallets/README.md) - from which the runtime is constructed.
+
+## Getting Started
+
+🦀 The template is using the Rust language.
+
+👉 Check the
+[Rust installation instructions](https://www.rust-lang.org/tools/install) for your system.
+
+🛠️ Depending on your operating system and Rust version, there might be additional
+packages required to compile this template - please take note of the Rust compiler output.
+
+### Build
+
+🔨 Use the following command to build the node without launching it:
+
+```sh
+cargo build --release
+```
+
+🐳 Alternatively, build the docker image:
+
+```sh
+docker build . -t polkadot-sdk-parachain-template
+```
+
+### Local Development Chain
+
+🧟 This project uses [Zombienet](https://github.com/paritytech/zombienet) to orchestrate the relaychain and parachain nodes. You can grab a [released binary](https://github.com/paritytech/zombienet/releases/latest) or use an [npm version](https://www.npmjs.com/package/@zombienet/cli).
+
+This template produces a parachain node. You still need a relaychain node - you can download the `polkadot` (and the accompanying `polkadot-prepare-worker` and `polkadot-execute-worker`) binaries from [Polkadot SDK releases](https://github.com/paritytech/polkadot-sdk/releases/latest).
+
+Make sure to bring the parachain node - as well as `polkadot`, `polkadot-prepare-worker`, `polkadot-execute-worker`, and `zombienet` - into `PATH` like so:
+
+```sh
+export PATH="./target/release/:$PATH"
+```
+
+This way, we can conveniently use them un the following steps.
+
+👥 The following command starts a local development chain, with a single relay chain node and a single parachain collator:
+
+```sh
+zombienet --provider native spawn ./zombienet.toml
+
+# Alternatively, the npm version:
+npx --yes @zombienet/cli --provider native spawn ./zombienet.toml
+```
+
+Development chains:
+
+- 🧹 Do not persist the state.
+- 💰 Are preconfigured with a genesis state that includes several prefunded development accounts.
+- 🧑‍⚖️ Development accounts are used as validators, collators, and `sudo` accounts.
+
+### Connect with the Polkadot-JS Apps Front-End
+
+🌐 You can interact with your local node using the
+hosted version of the Polkadot/Substrate Portal: [relay chain](https://polkadot.js.org/apps/#/explorer?rpc=ws://localhost:9944) and [parachain](https://polkadot.js.org/apps/#/explorer?rpc=ws://localhost:9988).
+
+🪐 A hosted version is also
+available on [IPFS](https://dotapps.io/).
+
+🧑‍🔧 You can also find the source code and instructions for hosting your own instance in the
+[`polkadot-js/apps`](https://github.com/polkadot-js/apps) repository.
+
+## Contributing
+
+🔄 This template is automatically updated after releases in the main [Polkadot SDK monorepo](https://github.com/paritytech/polkadot-sdk).
```

Comment:
I think it would look nicer if you be consistent with using or not using bullet points. 

I personally prefer such things to be bullet points.

Reply:
Done, I made it consistent with the bullet points.